testground/README.md

```diff
@@ -0,0 +1,62 @@
+# Guide
+
+[Testground documentation](https://docs.testground.ai/)
+
+## Getting started
+
+### Prerequisites
+
+- docker
+- go 1.22, or higher
+
+### Install Testground
+
+```bash
+$ git clone https://github.com/testground/testground.git
+$ cd testground
+# compile Testground and all related dependencies
+$ make install
+```
+
+It'll install the `testground` binary in your `$GOPATH/bin` directory, and build several docker images.
+
+### Running Testground
+
+```bash
+$ TESTGROUND_HOME=$PWD/data testground daemon
+```
+
+Keep the daemon process running during the test.
+
+### Running Test Plan
+
+Import the test plan before the first run:
+
+```bash
+$ TESTGROUND_HOME=$PWD/data testground plan import --from /path/to/cronos/testground/benchmark
+```
+
+Run the benchmark test plan locally:
+
+```
+$ testground run single --plan=benchmark --instances=2 --testcase=entrypoint --builder=docker:generic --runner=local:docker --wait
```

Comment:
do we need specify `TESTGROUND_HOME=$PWD/data` here

Reply:
No, it just communicate with the daemon

Reply:
for local:docker, do we need add Dockerfile like in [example](https://github.com/testground/testground/blob/master/plans/example-browser-node), I need run `TESTGROUND_HOME=$PWD/data testground run single --plan=example-browser-node --instances=2 --testcase=sync --builder=docker:generic --runner=local:docker --wait`

Reply:
we need to use `docker:nix` builder, supported by the patched testground linked in the PR description.